packages/node_modules/@ciscospark/redux-module-features/src/index.js

```diff
@@ -0,0 +1,45 @@
+import {Map} from 'immutable';
+
+export const ADD_FEATURE = `features/ADD_FEATURE`;
+
+export const initialState = new Map({
+  items: new Map()
+});
+
+export default function reducer(state = initialState, action) {
+  switch (action.type) {
+  case ADD_FEATURE:
+    return state.mergeIn([`items`], action.payload.feature);
+  default:
+    return state;
+  }
+}
+
+export function addFeature(feature) {
```

Comment:
Can we rename this to `storeFeature`? It took me a while to realize that this just saves the features we get from the server.